hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/snapshot/TestExportSnapshotNoCluster.java

```diff
@@ -67,29 +64,29 @@ public static void setUpBaseConf(Configuration conf) {
     conf.set(HConstants.HBASE_DIR, testDir.toString());
   }
 
-  @BeforeClass
-  public static void setUpBeforeClass() throws Exception {
+  @Before
+  public void setUpBefore() throws Exception {
     // Make sure testDir is on LocalFileSystem
-    testDir = TEST_UTIL.getDataTestDir().makeQualified(URI.create("file:///"), new Path("/"));
-    fs = testDir.getFileSystem(TEST_UTIL.getConfiguration());
+    testDir = testUtil.getDataTestDir().makeQualified(URI.create("file:///"), new Path("/"));
```

Comment:
Isolate more by getting method name into path?

Reply:
`testUtil.getDataTestDir()` will generate a path with a random UUID, so by getting a fresh instance of `testUtil`, I get an isolated path.